### PR TITLE
Rename playbackSpeed to playbackRate across the app

### PR DESCRIPTION
**Pull request by jhurliman**
**User-Facing Changes**
- None

**Description**
This changes the "playback speed" terminology in players to the more precise "playback rate". See 
<https://developer.mozilla.org/en-US/docs/Web/Guide/Audio_and_video_delivery/WebAudio_playbackRate_explained> for prior 
art.

### CODE REVIEW

**Comment by defunctzombie** [comment]
Counterpoint? http://docs.ros.org/en/diamondback/api/rostime/html/classros_1_1Rate.html

**Comment by defunctzombie** [comment]
From YouTube:

image

File: packages/studio-base/src/components/PlaybackRateControls.stories.tsx [hunk 1]
```diff
@@ -13,10 +13,10 @@
 import { storiesOf } from "@storybook/react";
 
 import MockMessagePipelineProvider from "@foxglove/studio-base/components/MessagePipeline/MockMessagePipelineProvider";
-import PlaybackSpeedControls from "@foxglove/studio-base/components/PlaybackSpeedControls";
+import PlaybackRateControls from "@foxglove/studio-base/components/PlaybackRateControls";
 import MockCurrentLayoutProvider from "@foxglove/studio-base/providers/CurrentLayoutProvider/MockCurrentLayoutProvider";
 
-const CAPABILITIES = ["setSpeed", "playbackControl"];
+const CAPABILITIES = ["setPlaybackRate", "playbackControl"];
```
Comment (jtbandes) [inline packages/studio-base/src/components/PlaybackRateControls.stories.tsx:19]: Do we have any players that dynamically get capabilities from some server – i.e. do we need to be backwards-compatible and continue to understand the `setSpeed` capability? Or is this purely used within the app itself?

**Comment by jtbandes** [comment]
lol w3schools 
image

**Comment by defunctzombie** [comment]
I am unconvinced we need to make this change. What problem does it solve?

**Comment by defunctzombie** [comment]
I think we can close this PR since we've determined that `playbackSpeed` is a reasonable name for this as well.

**Pull request closed without merging**